Makefile

```diff
@@ -24,7 +24,7 @@ GCC_GTEQ_450 := $(shell expr `${CC} -dumpversion | sed -e 's/\.\([0-9][0-9]\)/\1
 GCC_GTEQ_600 := $(shell expr `${CC} -dumpversion | sed -e 's/\.\([0-9][0-9]\)/\1/g' -e 's/\.\([0-9]\)/0\1/g' -e 's/^[0-9]\{3,4\}$$/&00/'` \>= 60000)
 GCC_GTEQ_700 := $(shell expr `${CC} -dumpversion | sed -e 's/\.\([0-9][0-9]\)/\1/g' -e 's/\.\([0-9]\)/0\1/g' -e 's/^[0-9]\{3,4\}$$/&00/'` \>= 70000)
 
-CFLAGS	+= -std=c99 -D__BSD_VISIBLE -D_ALL_SOURCE -D_XOPEN_SOURCE=600 -D_POSIX_C_SOURCE=200112 -D_ISOC99_SOURCE -D_REENTRANT -D_BSD_SOURCE -DVERSION=\"'$(VER)'\"
+CFLAGS	+= -std=c99 -D__BSD_VISIBLE -D_ALL_SOURCE -D_XOPEN_SOURCE=600 -D_POSIX_C_SOURCE=200112 -D_ISOC99_SOURCE -D_REENTRANT -D_BSD_SOURCE -D_DARWIN_C_SOURCE -DVERSION=\"'$(VER)'\"
```

Comment:
How about this instead:
```
    ifeq ($(OS),Darwin)
        CFLAGS += -D_DARWIN_C_SOURCE
    endif
```
Can you test if this works?
Then the macro would not be defined for the other operating systems.

Reply:
Yes, this works, but imo it is not worth adding this check since this macro is not used in platforms other than Darwin. Otherwise, in the same way we should check to be on FreeBSD for the macro `__BSD_VISIBLE`, I guess. Both macros extend the namespace for their specific platform.

Reply:
Ok, yes these macros are very specific and it should be ok to simply specify them for all platforms.